app/src/main/java/com/keylesspalace/tusky/db/AppDatabase.java

```diff
@@ -365,4 +366,31 @@ public void migrate(@NonNull SupportSQLiteDatabase database) {
             );
         }
     };
+
+    public static class Migration25_26 extends Migration {
+
+        private final File oldDraftDirectory;
+
+        public Migration25_26(File oldDraftDirectory) {
+            super(25, 26);
+            this.oldDraftDirectory = oldDraftDirectory;
+        }
+
+        @Override
+        public void migrate(@NonNull SupportSQLiteDatabase database) {
+            database.execSQL("DROP TABLE  `TootEntity`");
+
+            if (oldDraftDirectory != null && oldDraftDirectory.isDirectory()) {
+                File[] oldDraftFiles = oldDraftDirectory.listFiles();
+                if (oldDraftFiles != null) {
+                    for (File file : oldDraftFiles) {
+                        if (!file.isDirectory()) {
+                            file.delete();
```

Comment:
I don't know if this is the best way to delete those old files 🤔
it definitely works though

Reply:
I think its fine